evergreen/config.yml

```diff
@@ -229,7 +229,15 @@ functions:
           export TSAN_OPTIONS="suppressions=$(pwd)/test/tsan.suppress"
 
           cd build
-          $CTEST -C ${cmake_build_type|Debug} $TEST_FLAGS
+          if ! $CTEST -C ${cmake_build_type|Debug} $TEST_FLAGS; then
+            BAAS_PID=$(pgrep baas_server)
+            if [[ -n "$BAAS_PID" ]]; then
+              echo "Dumping baas to log file"
+              kill -3 $BAAS_PID
+              sleep 15
+            fi
```

Comment:
seems like this logic already exists here: https://github.com/realm/realm-core/blob/78175060de3787e6450005aed4cfb7072619a78e/evergreen/config.yml#L290-L308

maybe can call that function here?

Reply:
ah, okay feel free to ignore then